src/ds/logger.h

```diff
@@ -192,14 +275,25 @@ namespace logger
       // Sample: "2019-07-19 18:53:25.690267        "
```

Comment:
Should remove the right-padding spaces from this comment, since they're removed from `host_ts`.